pkg/appgw/backendhttpsettings.go

```diff
@@ -169,20 +169,18 @@ func (c *appGwConfigBuilder) getBackendsAndSettingsMap(cbCtx *ConfigBuilderConte
 	for backendID, serviceBackendPairs := range serviceBackendPairsMap {
 		if len(serviceBackendPairs) > 1 {
 			// more than one possible backend port exposed through ingress
-			e := controllererrors.NewErrorf(
-				controllererrors.ErrorMultipleServiceBackendPortBinding,
-				"service:port [%s:%s] has more than one service-backend port binding",
-				backendID.serviceKey(), backendID.Backend.ServicePort.String(),
-			)
-			c.recorder.Event(backendID.Ingress, v1.EventTypeWarning, events.ReasonPortResolutionError, e.Error())
-			glog.Warning(e.Error())
-			return nil, nil, nil, e
+			glog.V(5).Infof("service:port [%s:%s] has more than one service-backend port binding", backendID.serviceKey(), backendID.Backend.ServicePort.String())
```

Comment:
we should raise this as an error in the logs. Would make sense to let user know.
`glog.Errorf`